alerts/cluster-monitoring-operator/PrometheusKubernetesListWatchFailures.md

```diff
@@ -0,0 +1,81 @@
+# PrometheusKubernetesListWatchFailures
+
+## Meaning
+
+The `PrometheusKubernetesListWatchFailures` alert is triggered when `LIST/WATCH`
+requests to the Kubernetes API start failing during Prometheus target discovery.
+
+## Impact
+
+This might prevent Prometheus from adding, updating, or deleting targets effectively.
+
+## Diagnosis
+
+Determine whether the alert has triggered for the instance of Prometheus used
+for default cluster monitoring or for the instance that monitors user-defined
+projects by viewing the alert message's `namespace` label. The namespace for
```

Comment:
```suggestion
by checking the `namespace` label in  the alert message . The namespace for
```

Reply:
same wording uses in other runbooks https://github.com/search?q=repo%3Aopenshift%2Frunbooks+%22projects+by+viewing+the+alert+message%22&type=code